python/lsst/meas/astrom/matchPessimisticB.py

```diff
@@ -505,7 +503,6 @@ def _doMatch(self, refCat, sourceCat, wcs, refFluxField, numUsableSources,
         dist_cut_arcsec = np.max(
             (2 * wcs.pixelScale().asArcseconds(),
              clipped_dist_arcsec, maxMatchDistArcSec))
-        print("Dist cut", dist_cut_arcsec, "arcsec")
```

Comment:
If you think any of these print statements may be of interest to somebody then I strongly suggest you make these log statements at some suitable verbosity. INFO is usually printed. TRACE is the next step in verbosity, then DEBUG. So taking a guess at TRACE (and providing the data as a separate argument so the mess is only formatted if it is to be printed -- though the computation of `max` will presumably be more expensive than the formatting):

    self.log.trace("max distance = %s arcsec", distances_arcsec.max())